docs/guide/deployment/coap-gateway.md

```diff
@@ -137,8 +152,34 @@ CoAP API as specified in the [Open Connectivity Foundation - Device to Cloud Ser
 | `api.coap.tls.certFile` | string | `File path to certificate in PEM format.` | `""` |
 | `api.coap.tls.clientCertificateRequired` | bool | `If true, require client certificate.` | `true` |
 
+#### OAuth2.0 Client for UI and SDK
+
+Configured OAuth2.0 client is used by the mobile application or SDK to request a token used to authorize all calls they execute against other plgd APIs.
```

Comment:
It's used only for devices.

Reply:
`Configured OAuth2.0 client is used by devices to request a token used to authorize all calls they execute against other plgd APIs.`

?

Reply:
OAuth2.0 Client is used to issuing an authorization code used by the Onboarding Tool during the device provisioning which is then exchanged for an access token during the [SignUp operation](https://plgd.dev/architecture/component-overview/#cloud-registration)

Reply:
@Danielius1922 could you please update also plantumls in the docs? Adding OAuth2.0 Server against which the code is exchanged and add the CoAP gateway calling authorization-server AddDevice rpc call? Thanks!